### DISCUSSION TITLE
Persist theme configuration across sessions using local storage

### DISCUSSION THREAD

**Discussion by iamkyrylo** [primefaces/community#2222] [PrimeReact]
As a developer and user of the PrimeReact website, I find it inconvenient to select the needed theme configuration each time I visit the site, reload a page, or open a component documentation in a new tab. Storing the configuration in local storage would greatly improve the user experience by retaining the selected settings across sessions. Thank you for considering this feature!

**Comment by melloware** [comment]
Just a heads up it used to work this way and PrimeTek intentionally removed it: https://github.com/primefaces/primereact/issues/2671
